.golangci.yml

```diff
@@ -131,7 +131,7 @@ linters:
     - decorder
     # - depguard
     - dogsled
-    - dupl
+    # - dupl
```

Comment:
question: any reason you commented this out?

Reply:
Yeah, we would have needed a lot of nodupl like we did for the PG controller. We're duplicating a lot of code between PG, FW, and VPC controllers. A separate PR would be needed for using generics